website/content/docs/integrations/consul-connect.mdx

```diff
@@ -371,6 +371,30 @@ dashes (`-`) are converted to underscores (`_`) in environment variables so
 - Prior to Consul 1.9, the Envoy sidecar proxy will drop and stop accepting
   connections while the Nomad agent is restarting.
 
+## Troubleshooting
+
+If the sidecar service is not running correctly, you can investigate
+potential `envoy` failures in the following ways:
+
+* Task logs in the associated `connect-*` task
+* Task secrets (may contain sensitive information):
+  * envoy CLI command: `secrets/.envoy_bootstrap.cmd`
+  * environment variables: `secrets/.envoy_bootstrap.env`
+* An extra Allocation log file: `alloc/logs/envoy_bootstrap.stderr.0`
+
+For example, with an allocation ID starting with `b36a`:
+
+```shell-session
+nomad alloc status -short b36a  # to get the connect-* task name
+nomad alloc logs -task connect-proxy-count-api -stderr b36a
+nomad alloc exec -task connect-proxy-count-api b36a cat secrets/.envoy_bootstrap.cmd
+nomad alloc exec -task connect-proxy-count-api b36a cat secrets/.envoy_bootstrap.env
+nomad alloc fs b36a alloc/logs/envoy_bootstrap.stderr.0
+```
+
+Note: If the alloc is unable to start successfully, debugging files
+may only be accessible from the host filesystem.
```

Comment:
I think the `secrets` dir is mounted in a `tmpfs` in Linux, so they would not be available in the host.

Reply:
I went with this to soften the wording from the original commit message, which was

> If the alloc is unable to start successfully, the debugging files **are**
> only accessible from the host filesystem.

I debated leaving this note off entirely, but thought it might be useful for folks to consider, even if it doesn't provide a complete answer.  Would you suggest I mention secrets and tmpfs in particular here?

Reply:
I think a explicit referent that `secrets` may not be available at the host may be better, specially with this being a troubleshooting document, something went wrong so we should provide the most accurate information we can.

Maybe something like this? Kind of too long though 😅 

> Note: If the alloc is unable to start successfully, debugging files may only be accessible from the host filesystem. The sidecar task `secrets` directory may not be available in systems where it is mounted in a temporary file system.